### PR TITLE
181 get request attempt ids to results

### PR DESCRIPTION
**Pull request by dpmacisaac**
Let me know if everything looks good. I was moving things around a lot today and I think its readable but if there's anything weird lmk!

### CODE REVIEW

File: ComSemApp/templates/ComSemApp/student/assessment.html [hunk 1]
```diff
@@ -154,24 +168,37 @@ <h2 id="timer-text" class="show-expression"></h2>
...
+        $("#form").append(line);
+        resolve();
+      },
+      error: (jqXHR, textStatus, errorThrown) => {
+        cs_ajax_error(jqXHR, textStatus, errorThrown);
+        reject();
+      }
+    });
+    setTimeout(()=>{
+      console.log("timeout here");
```
Comment (aedmondson) [inline ComSemApp/templates/ComSemApp/student/assessment.html:198]: remove console.log

File: ComSemApp/templates/ComSemApp/student/assessment.html [hunk 2]
```diff
@@ -154,24 +168,37 @@ <h2 id="timer-text" class="show-expression"></h2>
...
+      reject();
```
Comment (aedmondson) [inline ComSemApp/templates/ComSemApp/student/assessment.html:199]: It might be useful to issue a cs_notification to let the user know it's timed out

**Review by aedmondson — CHANGES_REQUESTED** [review]
Just some minor things

**Comment by aedmondson** [comment]
resolves #181

File: ComSemApp/templates/ComSemApp/student/assessment.html (hunk 2 above)
Comment (dpmacisaac) [reply ComSemApp/templates/ComSemApp/student/assessment.html:199]: i decided to change this all together and use the ajax param "timeout" with 5000 ms as it works a bit better. It returns to the error field of the ajax if 5 secs goes by and it gives the text status "timeout" which is then thrown as a cs notification error

**Review by Jared-Torii — APPROVED** [review]
looks good, nice comments

**Pull request merged**